packages/cli/src/file-transformers/next-config-file-transformer.ts

```diff
@@ -3,70 +3,50 @@ import * as path from 'path';
 import * as recast from 'recast';
 import { parse as babelParse } from '@babel/parser';
 import type { namedTypes } from 'ast-types';
-import { builders } from 'ast-types';
 import type { MicroappFileTransformer } from '../file-transformer';
 import { CannotUpdateNextConfigFileError } from './errors';
 
 const TS_FILE_EXTENSION = '.ts';
-const MJ_SCRIPT_FILE_EXTENSION = '.mjs';
-const PLUGIN_MODULE = '@microapp-io/scripts';
-const PLUGIN_CLASS_NAME = 'MicroappNextFederationPlugin';
-const PLUGIN_IMPORT_CODE_ESM = `import { ${PLUGIN_CLASS_NAME} } from '${PLUGIN_MODULE}';\n`;
-const PLUGIN_IMPORT_CODE_CJS = `const { ${PLUGIN_CLASS_NAME} } = require('${PLUGIN_MODULE}');\n`;
-const CONFIG_KEY = 'config';
-const OPTIONS_KEY = 'options';
-const CONFIG_PLUGINS_KEY = `${CONFIG_KEY}.plugins`;
-const PLUGIN_USAGE_CODE = `if (!${OPTIONS_KEY}.isServer) {
-  ${CONFIG_PLUGINS_KEY}.push(new ${PLUGIN_CLASS_NAME}());
-}\n`;
-const CONFIG_WEBPACK_KEY = 'webpack';
 
 export class MicroappNextConfigFileTransformer
```

Comment:
We no longer need this Class as we don't have to modify the next.config file anymore.